### ISSUE TITLE
Cannot install ng2-dragula@1.1.7

### ISSUE THREAD

**Issue by vu-bui** [valor-software/ng2-dragula#249]
I got this error message while trying to install version 1.1.7
```
npm ERR! ng2-dragula@1.1.7 postinstall: `npm run webdriver-update && typings install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ng2-dragula@1.1.7 postinstall script 'npm run webdriver-update && typings install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ng2-dragula package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run webdriver-update && typings install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ng2-dragula
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls ng2-dragula
npm ERR! There is likely additional logging output above.
```
My `npm` version:
```
{ npm: '3.8.6',
  ares: '1.10.1-DEV',
  http_parser: '2.7.0',
  icu: '56.1',
  modules: '48',
  node: '6.1.0',
  openssl: '1.0.2h',
  uv: '1.9.0',
  v8: '5.0.71.35',
  zlib: '1.2.8' }
```

**Comment by lanocturne** [comment]
My 1.1.7 version's src files are missing
![image](https://cloud.githubusercontent.com/assets/4869206/15222913/a5b31d36-1826-11e6-9c1c-ae96f05f9842.png)

**Comment by yvesh** [comment]
Yep, same problem - src files are missing. Probably related to #239 

@vu-bui the webdriver-update problem can be solved by adding protractor and webdriver-manager to your dev-dependencies

**Comment by vu-bui** [comment]
Yeah, adding `protractor` and `webdriver-manager` made the error go away, but I suppose those shouldn't be in my project's dependencies. And I got the same missing source files problem too.

**Comment by haburka** [comment]
Using npm 3.8.6 and node 6.1.0, same issue

**Comment by haburka** [comment]
Try using versions 1.1.6 instead of 1.1.7 if you're having issues getting the source folder.

**Comment by vu-bui** [comment]
I was on 1.1.6 before, but it was built for angular2 beta releases, and I wanted to upgrade to the RC.

**Comment by lanocturne** [comment]
But 1.1.6 is not for RC1, right?

On Thu, May 12, 2016 at 11:36 AM haburka <notifications@github.com> wrote:

> Try using versions 1.1.6 instead of 1.1.7 if you're having issues getting
> the source folder.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/valor-software/ng2-dragula/issues/249#issuecomment-218846959>
>
-- 
Thanks,
Stanislaus

**Comment by yvesh** [comment]
No it's for beta, the patch from #239 is missing there. :(

**Comment by thomasbee** [comment]
Same for me, source files are missing.

**Comment by NathanWalker** [comment]
I'll get this republished and fixed in morning.

**Comment by valorkin** [comment]
Hey, Nathan after your fix I will update it to have same workflow as other ng2- modules, because I did npm version and publish, and it did not worked as expected :)

**Comment by valorkin** [comment]
fixed, last checks and I will publish

**Comment by valorkin** [comment]
Published to npm, please check

**Comment by jan-zajic** [comment]
npm package 1.1.8 does not contains ts or d.ts files, only generated js ones.. please can you fix it? other ng2 packages are right In this way (also previous dragula versions contains them)

**Comment by valorkin** [comment]
actually having only .js is expected behavior
not having .d.ts is an issue I will fix it

**Comment by valorkin** [comment]
.d.ts included in version 1.1.9 please check

**Comment by jan-zajic** [comment]
OK, .d.ts files are includes, thank you
